.github/workflows/release.yml

```
@@ -15,10 +15,10 @@ jobs:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2
         with:
-          version: latest
+          version: 7.30.3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18.15.0
```

Better to always get the latest in 18 IMO
```suggestion
          node-version: 18
```